tests/functional/setupVpn.js

```diff
@@ -150,7 +150,7 @@ exports.mochaHooks = {
 
   async afterEach() {
     // Collect errors on failure
-    if (this.currentTest.state === 'failed') {
+    if (!this.currentTest.state || this.currentTest.state === 'failed') {
```

Comment:
If the test fails at setup no logs are printed because there is no state. This fixes it.